### ISSUE TITLE
Boilerplate directory empty

### ISSUE THREAD

**Issue by axellebot** [angel-dart/cli#93]
Hello, I have a "FileException" and according to the output it's trying to copy a file that does not exists in my user directory :

<details><summary>CLI outputs</summary>
<p>

```
D:\Users\axell\Projects>angel init hello
Creating new Angel project in D:\Users\axell\Projects\hello...
Choose a project type before continuing:
Choose a project type before continuing:

1) Basic (Minimal starting point for Angel 2.x - A simple server with only a few additional packages.)
2) ORM (A starting point for applications that use Angel's ORM.)
3) GraphQL (A starting point for GraphQL API servers.)
4) Shared (Holds common models and files shared across multiple Dart projects.)
5) Shared (ORM) (Holds common models and files shared across multiple Dart projects.)


✔ Enter your choice 2
Cloning "ORM" boilerplate from "https://github.com/angel-dart/angel.git"...
$ git clone --depth 1 --single-branch -b orm https://github.com/angel-dart/angel.git C:\Users\axell\.angel\boilerplates\angel.orm
Cloning into 'C:\Users\axell\.angel\boilerplates\angel.orm'...
remote: Enumerating objects: 57, done.
remote: Counting objects: 100% (57/57), done.
remote: Compressing objects: 100% (48/48), done.
remote: Total 57 (delta 2), reused 35 (delta 1), pack-reused 0
Unpacking objects: 100% (57/57), done.
✗ Could not initialize Angel project.
Oops, something went wrong: FileSystemException: Cannot copy file to 'hello\.dockerignore', path = 'C:\Users\axell\.angel\boilerplates\angel.orm\.dockerignore' (OS Error: The system cannot find the path specified.
, errno = 3)
```

</p>
</details>

<details><summary>Configuration</summary>
<p>

Windows 10

2 Hard drive (`C:`, `D:`)

</p>
</details>

Indeed `C:\Users\axell\.angel\boilerplates` appear to be totally empty.

I tried to `deactivate` and `activate` the CLI to possibly reinstall the boilerplate content.

**Comment by thosakwe** [comment]
I think I've encountered the same error, but can you paste the output?

**Comment by rosierjolie** [comment]
I am also running into the same error.

**Comment by thosakwe** [comment]
Resolved version `2.1.7.+1`: https://pub.dev/packages/angel_cli/versions/2.1.7+1
